test/azure/generated/AzureCompositeModelClient/azureCompositeModel.ts

```diff
@@ -144,17 +144,16 @@ class AzureCompositeModel extends msRestAzure.AzureServiceClient {
 
     // Create HTTP transport objects
     const httpRequest = new WebResource();
-    // Send Request
     let operationRes: msRest.HttpOperationResponse;
     try {
       operationRes = await client.sendOperationRequest(
         httpRequest,
         {
           arguments: {
             "this.subscriptionId": this.subscriptionId,
-            "resourceGroupName": resourceGroupName,
-            "apiVersion": apiVersion,
-            "this.acceptLanguage": this.acceptLanguage,
+            resourceGroupName: resourceGroupName,
```

Comment:
Probably not for this PR, but one to keep in your back pocket: is it possible to generate merely `{ resourceGroupName }` instead of `{ resourceGroupName: resourceGroupName }` when the expression is a valid identifier and equal to the key? The space savings could add up.

Reply:
That's definitely possible. Let me see if I can get it done here.